app/docker/components/imageRegistry/por-image-registry.html

```diff
@@ -6,7 +6,7 @@
     </label>
     <div ng-class="$ctrl.inputClass">
       <select
-        ng-options="registry as registry.Name for registry in $ctrl.registries track by registry.Id"
+        ng-options="registry as registry.Name for registry in $ctrl.registries track by $index"
```

Comment:
how does this help? `track by` is similar to react's `key` prop. it doesn't control sort order. if you want to sort you need to do it on the controller